src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.UnknownOperationException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopService {
+    private Map<String, Integer> fruitInventory = new HashMap<>();
```

Comment:
Consider following the encapsulation principle more strictly. The 'fruitInventory' should likely be private to prevent direct access and modification from outside the class. If modification is required, consider providing more controlled methods to handle it.